common-rendering/src/editorialPalette/hover.ts

```diff
@@ -33,10 +34,15 @@ const pagination = (format: ArticleFormat): Colour => {
 	}
 };
 
+const signUpFormButton = (_format: ArticleFormat): Colour => neutral[7];
+const signUpFormButtonDark = (_format: ArticleFormat): Colour => '#d1d1d1';
```

Comment:
Do you know why we're not able to use a Source colour for this? Is that a question for Design?